.devcontainer/devcontainer.json

```
@@ -22,7 +22,7 @@
   "mounts": [
     {
       "source": "devcontainer-cargo-cache",
```

```suggestion
      "source": "devcontainer-cargo-cache-${devcontainerId}",
```

As pointed out in devcontainers/templates#117, this seemed necessary.

---

Ah thanks!